### ISSUE TITLE
When installing/upgrading multiple packages, download installers in parallel.

### ISSUE THREAD

**Issue by legowerewolf** [microsoft/winget-cli#3117]
### Description of the new feature / enhancement

Some downloads are slow. Many downloads at once can take a lot of time, especially if they have to wait on each other. Download installers in parallel, then execute them in series.

**Comment by denelon** [comment]
Duplicate of #225
